docs/guide/repository.md

```diff
@@ -130,7 +130,7 @@ Technically, a Section is simply a triplet: a number, a name and an identifier.
 
 When a new Content item is created, its Section ID is set to the default Section (which is usually Standard). When the item is published it is assigned to the same Section as its parent. Because Content must always be in a Section, unassigning happens by choosing a different Section to move it into. If a Content item has multiple Location assignments then it is always the Section ID of the item referenced by the parent of the main Location that will be used. In addition, if the main Location of a Content item with multiple Location assignments is changed then the Section ID of that item will be updated.
 
-When content is moved to a different Location, the item itself and all of its subtree will be assigned to the Section of the new Location.
+When content is moved to a different Location, the item itself and all of its subtree will be assigned to the Section of the new Location. Please note that it works only for copy and move; updating parents' Location does not affect the subtree, meaning that Content Location cannot be updated this way.
```

Comment:
`updating parents' Location does not affect the subtree, meaning that Content Location cannot be updated this way.`
=>
`assigning a new section to a parents Content does not affect the subtree, meaning that Subtree cannot currently be updated this way _(yet)_.`